aws-frauddetector-detector/aws-frauddetector-detector.json

```diff
@@ -457,6 +457,9 @@
     "/properties/Rules/*/Outcomes/*/Arn",
     "/properties/Rules/*/Outcomes/*/CreatedTime",
     "/properties/Rules/*/Outcomes/*/LastUpdatedTime",
+    "/properties/EventType/Arn",
+    "/properties/EventType/CreatedTime",
+    "/properties/EventType/LastUpdatedTime",
```

Comment:
We had marked event type dependency arn/created/lastupdated as readonly, but we also need to mark the same for event type, otherwise contract tests complain when the output model has additional fields.